src/theory/sets/theory_sets_private.cpp

```diff
@@ -1292,6 +1292,7 @@ bool TheorySetsPrivate::collectModelInfo(TheoryModel* m)
     else
     {
       std::vector<Node> els;
+      std::set<Node> elsSet;
```

Comment:
Why is this local variable declared here? It is unused.

Reply:
Will delete.